api/api_public_transaction_pool.go

```diff
@@ -502,7 +502,13 @@ func (s *PublicTransactionPoolAPI) PendingTransactions() ([]map[string]interface
 // Resend accepts an existing transaction and a new gas price and limit. It will remove
 // the given transaction from the pool and reinsert it with the new gas price and limit.
 func (s *PublicTransactionPoolAPI) Resend(ctx context.Context, sendArgs SendTxArgs, gasPrice *hexutil.Big, gasLimit *hexutil.Uint64) (common.Hash, error) {
-	if sendArgs.AccountNonce == nil {
+	return resend(s, ctx, &sendArgs, gasPrice, gasLimit)
+}
+
+// Resend accepts an existing transaction and a new gas price and limit. It will remove
+// the given transaction from the pool and reinsert it with the new gas price and limit.
```

Comment:
The description of this function says that it removes the existing transaction.
But it doesn't seem to be removed, only reinsert.

Reply:
@nohkwak Good question. This API is supposed to be used to adjust gasPrice of existing transactions. The API finally calls `s.b.SendTx()`, and the new tx will be validated again. If the new gas price is bigger than the gas price of txpool tx, then it will replace the exiting tx.

Reply:
I agree with you.
I also found a few mistake in formal comments in api files.
Let me fix all things in api refactoring PR after this